packages/utils/package.json

```diff
@@ -1,9 +1,51 @@
 {
   "name": "@element-plus/utils",
   "private": true,
+  "homepage": "https://github.com/element-plus/element-plus#readme",
+  "bugs": {
+    "url": "https://github.com/element-plus/element-plus/issues"
+  },
   "license": "MIT",
-  "main": "index.ts",
-  "peerDependencies": {
+  "author": "Element Plus Team <hello@element-plus.org>",
+  "files": [
+    "dist"
+  ],
+  "main": "./index.ts",
+  "module": "./index.ts",
+  "unpkg": "index.js",
+  "jsdelivr": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/element-plus/element-plus.git",
+    "directory": "packages/utils"
+  },
+  "scripts": {
+    "clean": "rimraf dist",
+    "build": "tsup ./*.ts --dts",
+    "dev": "tsup ./*.ts --watch",
+    "prepack": "clean-package",
+    "postpack": "clean-package restore",
+    "typecheck": "tsc --noEmit"
+  },
+  "clean-package": "../../clean-package.json",
+  "dependencies": {
+    "@vueuse/core": "^9.1.0",
     "vue": "^3.2.0"
+  },
+  "devDependencies": {
+    "@types/node": "^18.19.25",
+    "clean-package": "^2.2.0",
+    "tsup": "^8.1.0"
+  },
+  "tsup": {
```

Comment:
I recommend creating `tsup.config.ts` to manage the build configuration, which will have type hints.

Reply:
Will do, the next upcoming PR will be extracting the common parts.